outsite.nl/pages/index.vue

```diff
@@ -61,12 +61,12 @@ const instagramChannels = [
   </LayoutHeader>
 
   <section class="overflow-x-hidden overflow-y-visible pt-48 -mt-48 -mb-40">
-    <div class="container mx-auto pt-12 sm:pt-16 pb-24 md:flex md:justify-between">
+    <div class="container mx-auto pt-12 sm:pt-16 pb-28 md:flex md:justify-between">
       <div class="md:w-1/2 px-4">
         <PagesHomeIntro />
       </div>
       <div class="md:w-2/5 overflow-hidden md:overflow-visible relative">
-        <Panda class="relative md:absolute bottom-0 left-5 w-full" />
+        <Panda class="relative md:absolute bottom-0 left-5 w-full rotate-[12deg]" />
```

Comment:
Instead of rotating the panda, I think it's better to fix the positioning for 430px-768px screens (not necessarily with such a specific target). Just to not change the visual appearance of the panda ;) 🐼

Reply:
Hey, I tried shifting the panda down before rotating it. It looked a bit odd, like just a stub peeking out. I wasn't happy with that, so I said just aligned the image with the diagonal stripe; it was more consistent, looked similar at all resolutions, and we see more of the panda. 

Ill try putting up a screenshot with the repositioned panda for comparison later today.